content/work/adobe.md

```diff
@@ -147,15 +147,60 @@ summary: |
   responsive, accessible design.
 ---
 
-{% import 'quotes.macros.njk' as quotes %}
 {% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
+{% import 'quotes.macros.njk' as quotes %}
 
 At the end of the game, players discover their own typographic superpower.
 Players have the option to share their superpower on social media, access the
 typeface on Adobe, view similar typefaces, and download their superpower badge.
 
 {{ quotes.grid(press) }}
 
+{{ layout.title('What We Did') }}
+
+{% call embed.media_block(
+  media=embed.screen(
+    notch='0',
```

Comment:
This looks broken, since the notch is covering a form control.

Reply:
(the same is true on CFO share, but I guess that's not part of this PR?)

Reply:
I don't know what notch is or how it works. Could you provide a screenshot of what is looking broken? Whatever is breaking may be a result of me copying and pasting code that was meant to do something special for a specific work detail page.

Reply:
`notch` provides an offset for the iphone notch that’s covering up some of our designs, because `notch: 0` is removing that space. The default should be right in most cases, if you remove this line entirely. There's also a `fill` option, for saying what color should be used to fill that space.